src/mantine-core/src/Rating/Rating.styles.ts

```diff
@@ -0,0 +1,92 @@
+import { createStyles, MantineSize } from '@mantine/core';
+import { CSSObject } from '@mantine/styles/src/tss/types/css-object';
+
+export interface RatingStylesParams {
+  size: MantineSize;
+  readonly: boolean;
+}
+
+export default createStyles((theme, { readonly }: RatingStylesParams, getRef) => {
+  const visuallyHidden: CSSObject = {
```

Comment:
Why it cannot be hidden with `display:none` or `visibility:hidden`?

Reply:
with `display: none` on input, focus will not work.

with `visibility: hidden`, elements take up the space, which is undesirable